src/Paramore.Brighter.MessagingGateway.Kafka/KafkaMessageConsumer.cs

```diff
@@ -167,7 +167,7 @@ public class KafkaMessageConsumer : KafkaMessagingGateway, IAmAMessageConsumer
                 })
                 .SetPartitionsRevokedHandler((consumer, list) =>
                 {
-                    _consumer.Commit(list);
+                    consumer.Commit(list);
```

Comment:
Presumably this was incorrectly capturing the field in the class rather than using the provided `consumer`